install/kubernetes/helm/istio/charts/tracing/templates/deployment-jaeger.yaml

```diff
@@ -23,6 +23,11 @@ spec:
         scheduler.alpha.kubernetes.io/critical-pod: ""
         prometheus.io/scrape: "true"
         prometheus.io/port: "16686"
+{{- if .Values.contextPath }}
+        prometheus.io/path: "{{ .Values.contextPath }}/metrics"
+{{- else }}
+        prometheus.io/path: "/{{ .Values.provider }}/metrics"
+{{- end }}
```

Comment:
LGTM.
Only question is whether `.Values.provider` is required in else. If context path is not set then I believe Jaeger would use endpoint `/metrics`?

Reply:
it has to match the `QUERY_BASE_PATH`, which is set to `/{{ .Values.provider }}` later in the template.

Reply:
Ah ok thanks.